stopify-continuations/bin/build

```diff
@@ -5,10 +5,16 @@ set -e
 
 mkdir -p dist/tmp
 
+# Generate pre-stopified versions of each runtime
+
+echo 'Generating pre-stopififed runtimes'
+
 for TRANSFORM in lazy eager retval fudge; do
   ./bin/stopify-continuations.js -t $TRANSFORM dist/src/runtime/implicitApps.js dist/tmp/implicitApps.$TRANSFORM.js
   ./bin/stopify-continuations.js -t $TRANSFORM dist/src/runtime/hofs.js dist/tmp/hofs.$TRANSFORM.js
-  ./bin/stopify-continuations.js -t $TRANSFORM dist/src/runtime/gettersSetters.js dist/tmp/gettersSetters.$TRANSFORM.js
```

Comment:
Why is this okay to do? What if a getter captures a stack?

Reply:
Getters runtime has no function calls. All functions are trivially flat.

Reply:
You're right. I guess I'm confused how getter work at all.